http/server.ts

```diff
@@ -122,6 +126,26 @@ export class ServerRequest {
   conn: Conn;
   r: BufReader;
   w: BufWriter;
+  private _cookie: Cookie;
+
+  get cookie(): Cookie {
+    if (this._cookie) {
+      return this._cookie;
+    }
+    if (this.headers.has("Cookie")) {
+      const out: Cookie = {};
+      const c = this.headers.get("Cookie").split(";");
```

Comment:
```suggestion
      const c = this.headers.get("cookie").split(";");
```

Reply:
Cases are ignored in the HTTP specification, and `deno` is also parsed this way.

Although the results of both the `Cookie` and the `cookie` are the same

Reply:
Yes it's ignored but in the specs it's written `Cookie` that's why i've corrected this.